[15.0][ADD] partner_delivery_link (#1271)
OCA/partner-contact: partner_delivery_link/readme/DESCRIPTION.rst

> Partner Delivery Link module is a part of partner-contact and count the totals picking
> created for customer in customer form and can access all pickings.

Reviewer: ```suggestion
Partner Delivery Link module counts the total pickings
created for customer and adds button (on customer form) to access them.
```

something like that
